tiled/server/authentication.py

```diff
@@ -503,7 +526,7 @@ async def create_tokens_from_session(settings, db, session, provider):
     }
 
 
-def build_auth_code_route(authenticator, provider):
+def build_auth_code_route(authenticator: ExternalAuthenticator, provider: str):
```

Comment:
all the inner `route` functions in the  `build_..._routes` could be called directly from `add_external_authenticator_routes` now

Reply:
done this